src/Vimeo/Vimeo.php

```diff
@@ -142,6 +143,13 @@ private function _request($url, $curl_opts = array()) {
         curl_setopt_array($curl, $curl_opts);
         $response = curl_exec($curl);
         $curl_info = curl_getinfo($curl);
+
+        if(isset($curl_info['http_code']) && $curl_info['http_code'] === 0){
+            $curl_error = curl_error($curl);
+            $curl_error = !empty($curl_error) ? '[' . $curl_error .']' : $curl_error;
```

Comment:
The right hand of this ternary should just be an empty string `''`. If curl_error returns false this will end up with a weird message